### PR TITLE
Allows to pass the settings.json file location via the --settings argument

### PR DESCRIPTION
**Pull request by ahorn42**
Allows to pass the settings.json file location via the --settings argument as a path in addition to passing its content.

Also updated documentation, which did not list the other loading possibilities added with #715.

### CODE REVIEW

File: Unreal/Plugins/AirSim/Source/SimHUD/SimHUD.cpp [hunk 1]
```diff
@@ -372,6 +373,11 @@ bool ASimHUD::getSettingsTextFromCommandLine(std::string& settingsText)
         FString commandLineArgsFString = FString(commandLineArgs);
         int idx = commandLineArgsFString.Find(TEXT("-settings"));
         FString settingsJsonFString = commandLineArgsFString.RightChop(idx + 10);
+
+		if (readSettingsTextFromFile(settingsJsonFString.TrimQuotes(), settingsText)) {
+			return true;
+		}
+
```
Comment (rajat2004) [inline Unreal/Plugins/AirSim/Source/SimHUD/SimHUD.cpp:380]: Please change the indentation from Tabs -> 4 spaces as is followed in the file

File: docs/settings.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
 # AirSim Settings
 
 ## Where are Settings Stored?
-Windows: `Documents\AirSim`
-Linux: `~/Documents/AirSim`
+AirSim is searching for the settings definition in 4 different ways in the following order. The first match will be used:
 
-The file is in usual [json format](https://en.wikipedia.org/wiki/JSON). On first startup AirSim would create `settings.json` file with no settings. To avoid problems, always use ASCII format to save json file.
+1. Looking at the path specified by the `--settings` command line argument.
```
Comment (rajat2004) [inline docs/settings.md:6]: This requires the full path of the settings file correct? Mentioning that and adding an example command like in the comments above will be great

**Review by rajat2004 — COMMENTED** [review]
Wasn't aware of this, thanks!

File: Unreal/Plugins/AirSim/Source/SimHUD/SimHUD.cpp (hunk 1 above)
Comment (ahorn42) [reply Unreal/Plugins/AirSim/Source/SimHUD/SimHUD.cpp:380]: changed it ;)

File: docs/settings.md (hunk 2 above)
Comment (ahorn42) [reply docs/settings.md]: added an example and a remark that it must be an absolute path

**Comment by ahorn42** [comment]
just updated the pull request according to your suggestions :)

**Comment by rajat2004** [comment]
Thanks :) One last thing which will be good to do would be to squash the commits into 1, that keeps the history cleaner and makes life easy for the maintainers

**Comment by ahorn42** [comment]
@rajat2004 done ;)

**Review by TomSeestern — APPROVED** [review]

**Pull request merged**